add_provider.yml

```diff
@@ -10,8 +10,8 @@
       miq_url: '{{ miq_url | default(omit) }}'
       miq_username: '{{ miq_username | default(omit) }}'
       miq_password: '{{ miq_password | default(omit) }}'
-      hostname: '{{ hostname }}'
-      port: '{{ port }}'
+      ems_api_hostname: '{{ ems_api_hostname }}'
+      ems_api_port: '{{ ems_api_port | default(omit) }}'
       token: '{{ token }}'
```

Comment:
@dkorn shouldn't we follow the same naming for these as well? `ems_auth_token`, `ems_hawkular_hostname`, `ems_hawkular_port`, etc.